tests/lca/imagebasedupgrade/mgmt/negative/tests/ibu-invalid-next-stage.go

```diff
@@ -0,0 +1,76 @@
+package negative_test
+
+import (
+	"time"
+
+	. "github.com/onsi/ginkgo/v2"
+	. "github.com/onsi/gomega"
+	"github.com/openshift-kni/eco-goinfra/pkg/lca"
+	"github.com/openshift-kni/eco-goinfra/pkg/reportxml"
+	"github.com/openshift-kni/eco-gotests/tests/lca/imagebasedupgrade/internal/nodestate"
+	. "github.com/openshift-kni/eco-gotests/tests/lca/imagebasedupgrade/mgmt/internal/mgmtinittools"
+	"github.com/openshift-kni/eco-gotests/tests/lca/imagebasedupgrade/mgmt/negative/internal/tsparams"
+	"golang.org/x/exp/slices"
+)
+
+var _ = Describe(
+	"Patching ibu stage from Idle to Rollback",
+	Ordered,
+	Label(tsparams.LabelImmutableSeedImage), func() {
+		var (
+			ibu *lca.ImageBasedUpgradeBuilder
+			err error
+		)
+
+		BeforeAll(func() {
+			By("Pull the imagebasedupgrade from the cluster")
+			ibu, err = lca.PullImageBasedUpgrade(APIClient)
+			Expect(err).NotTo(HaveOccurred(), "error pulling ibu resource from cluster")
+
+			By("Ensure that imagebasedupgrade stage is set to Idle")
+			Expect(string(ibu.Object.Spec.Stage)).To(Equal("Idle"), "error: ibu resource contains unexpected state")
```

Comment:
Shouldn't you set the seed image and seed image version here? Otherwise you might run into unrelated issues.

Reply:
Updated.